### DISCUSSION TITLE
Taro + react 生成的支付宝小程序, 鸿蒙系统打开白屏？

### DISCUSSION THREAD

**Discussion by louwenqiang** [NervJS/taro#14542] [Q & A]
打包生成的支付宝小程序，ios和安卓机扫码可以正常打开，鸿蒙系统打开小程序显示白屏，有人遇到过吗，按理鸿蒙系统打开其他支付宝小程序是正常的
![image](https://github.com/NervJS/taro/assets/34569315/4e5f9a68-dad0-4c57-85d4-1df7f2f6f852)

**Comment by bigmeow** [comment]
你最好贴个错误，一般白屏都是报错了

**Reply by louwenqiang** [reply]
正好借了个鸿蒙的在真机调试，已贴
